src/Service/BookingCodes.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace CommonsBooking\Service;
+
+use CommonsBooking\Messages\BookingCodesMessage;
+use CommonsBooking\Model\BookingCode;
+
+use WP_Query;
+
+class BookingCodes {
+
+
+	/**
+	 * Send Booking Codes by E-mail.
+     * is triggered in  Service\Scheduler initHooks()
+	 * 
+	 */
+	public static function sendBookingCodesMessage() {
+		$query = new WP_Query( [
+			'post_type'    => 'cb_timeframe',
```

Comment:
Würde das auch hier lieber durch die $postType Variable von dem Timeframe ersetzen (in CustomPostType/Timeframe)